File: complementary_DNA/complementary_DNA.rb
def DNA_strand(dna)
  dna_arr = []
  dna_arr = dna.split('')
  complementary_arr = []
  dna_arr.each do | nucleotide |
    if nucleotide == 'A'
      complementary_arr.push('T')
    elsif nucleotide == 'T'
      complementary_arr.push('A')
    elsif nucleotide == 'C'
      complementary_arr.push('G')
    else complementary_arr.push('C')
    end
  end
  complementary_arr
  complementary = complementary_arr.join
end

File: complementary_DNA2/complementary_DNA.js
function DNAStrand(dna){
  dna.split('');
  let output = [];
  for (let i=0 ; i < dna.length; i++) {
    if (dna[i] == 'A') {
      output.push('T');
      }
    else if (dna[i] == 'C') {
      output.push('G');
      }
    else if (dna[i] == 'G') {
      output.push('C');
      }
    else {
      output.push('A');
    }
  }
  let outputstr = output.join('');
  console.log(outputstr);
  return outputstr;
}
